content/material_and_questions.md

```diff
@@ -44,4 +53,151 @@ In the below table, any minimum or maximum variation between material years are
 | Finish          |       3 |       4 | FT, FTN, FTV, F2V    |
 | Situation       |     2/0 |     4/0 | SIT                  |
 
+Question Type Minimums
+: For a given Question Type, the minimum number of questions must be asked in Questions Numbered 1-20. This excludes any questions appended with "A" or "B."
+
+Question Type Maximums
+: For a given Question Type, the maximum number of questions must not be exceeded in all "non-Overtime" questions. This includes any questions appended with "A" or "B."
+
+Overtime Specifics
+Question Type maximums can be exceeded in Overtime.
+
+## Question Writing
+
+### Preparation of Questions
+All questions must contain verbatim material. On some types, the addition of a single, approved, interrogative word at the very beginning or very end is required.
+
+### Question Types
+
+#### Interrogative
+Should not be overly long and should be clear and precise.
+Must have exactly one inserted interrogative word.
+Must have a single answer.
+Must include a unique word, or 2-3 word unique phrase within the first 5 words of the question. 
+The Question Type abbreviation is "INT"
+
+#### Multiple Answer
+Must have exactly one inserted interrogative word.
+Must have more than one answer.
+- The clarification of a single answer does not meet the above requirement.
+- The identification of a singular pronoun does not meet the above requirement.
+- The identification of a plural pronoun does meet the above requirement.
+Must include a unique word, or 2-3 word unique phrase within the first 5 words of the question. 
+If the multiple answers contain at least one "positive" answer AND at least one "negative" answer, this is deemed invalid as a Multiple Answer question.
+The Question Type abbreviation is "MA"
+
+#### Reference
+Used to distinguish exact same words or phrases from the material.
+The entire question is part of the required question + answer.
+If a quizzer needs to provide the reference question, the provided reference question must: 
+- Be of the exact same type as the one on the card (CR, CVR, CRMA, CVRMA).  The quizmaster and/or answer judge must check to confirm this.
+- Have content AND wording is similar to the content and wording of the written question.
+    - For greater clarity, we have provided the following example: CVR According to Luke chapter 23: 37 King of what?  A. The Jews The quizzer can give the question “King of what?” or “King of whom?”, as Jews could be referring to either nation (what) or people (whom).  In this case the quizzer could also give “King of the what?”
+- Have the determining word or phrase from the quizmaster’s card. 
+    - In the 50/50 example, “What men?”, and “Lawless what?, the questions are not similar.
+The determination of a valid Reference question, and the valid sub-type only considers the content in the question. 
```

Comment:
this is also a re-write by me, but I think it maintains functional parity.

Reply:
I'm actually not sure what this rule means as written. Is this meant to address the issue where some districts think that a different interrogative word (what instead of who) can make the question "unique" even though the non-interrogative content is identical to something else in the chapter?

Reply:
It's making the wording and application more explicitly general of this, which is currently under the MA section under Refs. "The exact word or phrase appearing elsewhere does not necessarily have to be a multiple answer to write this question type."

Reply:
Examples should all be under an "Example" header.

Reply:
Can you help me out with this?

Since it's underneath a 4 header, do I use a 5 header?

and if I do that, what happens to the line starting with "the determination of a valid Reference..." ??

Reply:
Yup. Whatever header you're under, just add 1. You'll want the example to be after the end of the current section of content, though. There isn't a good way in Markdown to say "go back up 1 level and continue". So just put examples at the end of the section.

Reply:
Is there a good way to denote which specific line/bullet/sentence, the example refers to? I guess I'm asking about footnotes, kinda.

Reply:
Not easily, no. Just have to do it with language.

Reply:
LOL, this example reads so stupid now, "Referencing the determining word or phrase, in the 50/50 example, “What men?”, and “Lawless what?, the questions are not similar."

the example says "not similar" --it doesn't say "does not have the determining word" -- it basically ISN'T an example at all. ugh.

Reply:
@jswingle can you help with a reword of "The determination of a valid Reference question, and the valid sub-type only considers the content in the question." ?